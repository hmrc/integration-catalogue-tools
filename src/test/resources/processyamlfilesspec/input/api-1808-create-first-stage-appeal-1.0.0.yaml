openapi: 3.0.3
info:
  title: 'API#1808 Create First Stage Appeal'
  description: >
        This API allows a customer to create a first stage appeal against a late submission penalty/ penalty point by providing reasonable excuse and date for a penalty id.
  version: 1.0.0
  contact:
    name: Denin Thomas
    email: denin.thomas@hmrc.gov.uk

tags:
  - name: API#1808
    description: Create First Stage Appeal
