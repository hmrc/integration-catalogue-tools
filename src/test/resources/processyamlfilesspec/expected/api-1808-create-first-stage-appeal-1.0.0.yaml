openapi: 3.0.3
info:
  title: 'API#1808 Create First Stage Appeal'
  description: >
        This API allows a customer to create a first stage appeal against a late submission penalty/ penalty point by providing reasonable excuse and date for a penalty id.
  version: 1.0.0
  contact:
    name: Denin Thomas
    email: denin.thomas@hmrc.gov.uk
  x-integration-catalogue:
    reviewed-date: 2022-04-22T20:27:05Z
    platform: CORE_IF
    publisher-reference: 1808

tags:
  - name: API#1808
    description: Create First Stage Appeal
